### PR TITLE
info

### PR DESCRIPTION
**Pull request by schvetsov**

### CODE REVIEW

File: src/containers/GetList.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+  constructor(props) {
+    super(props);
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  componentDidMount() {
+    return axios.get('https://fb-assessment.glitch.me/artists')
+      .then(res => this.props.dispatch(updateList(res.data)))
+      .catch(err => console.log(err))
```
Comment (schvetsov) [inline src/containers/GetList.js:18]: Should I move this out to a separate file?

**Review by schvetsov — COMMENTED** [review]
questions

**Comment by MichaelDimmitt** [comment]
@schvetsov, you should close this pr you can still have conversation with a closed pr.

**Comment by schvetsov** [comment]
Made proposed changes

**Pull request merged**